__mocks__/mail_service.js

```diff
@@ -0,0 +1,11 @@
+
+const mailService = {
+    addMoreSlotsHosts: async (host, hosts, organizer, guest) => null,
+
+    askForMoreSlots: async (guest, hosts) => null,
+
+    pickSlotGuest: async (guest, meetingUuid, organizer) => null
```

Comment:
Jeśli chcesz pokazać, że to pusta funkcja, to możesz zrobić () => {} (trochę bardziej jsowo), chyba że ona faktycznie ma zwracać nulla, to wtedy mój komentarz jest zbędny.

Reply:
Wolę dostać nulla niż undefined XD